src/components/SelectedCategoryPage/useSelectedCategoryPage.ts

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import { useSelector, useDispatch } from "react-redux";
+import {useParams,useNavigate} from 'react-router-dom'
+import {useEffect} from 'react'
+import {actions,selectorsCategory} from 'store/sliceCategory'
+import { AppDispatch } from 'store/store';
```

Comment:
```suggestion
import { AppDispatch } from 'store';
```
тянем из Index файла